west.yml

```diff
@@ -96,11 +100,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 705d00a71149b876abbbe681631be0b89de83334
-    - name: hal_nordic
-      repo-path: sdk-hal_nordic
-      path: modules/hal/nordic
-      revision: ef6c7c3e82fee6e9e58c5295a4de7201b4b99b3c
+      revision: radio_16_support
```

Comment:
we can replace this one with `pull/203/head`

Reply:
https://github.com/nrfconnect/sdk-nrf/commit/829684b16a22e7a3fbbf9f0a060853c2ccb0b33a

Done already